### PR TITLE
🔧🏗 Setup project with hmpps template

### PR DESCRIPTION
**Pull request by astroash**
- initial commit take 2

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: "3"
+services:
+  offender-assessment-events:
+    build:
+      context: .
+    network_mode: "host"
+    container_name: offender-assessment-events
+    ports:
+      - "8081:8080"
```
Comment (nickmcmahon01) [inline docker-compose.yml:9]: 8080:8080 or you did you not change this from the default?

Comment (astroash) [reply docker-compose.yml:9]: Did'nt change from default

Comment (astroash) [reply docker-compose.yml:9]: As it's a template seems to be easiest to merge as is, then make any changes we want in subsequent PRs. This way the setup is easily documented in git history so we can replicated for the other services

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+image:
+  repository: quay.io/hmpps/offender-assessment-events
+  tag: latest
+  port: 8080
+
+ingress:
+  enabled: true
+  enable_whitelist: true
+  hosts:
+    - host: offender-assessment-events-dev.hmpps.service.justice.gov.uk
```
Comment (nickmcmahon01) [inline helm_deploy/values-dev.yaml:15]: This needs to be different for the AKS cluster, it won't be this address, and the name offernder-assessment-events-dev will be too long for the 64 char limit.

File: helm_deploy/values-dev.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
```
Comment (nickmcmahon01) [inline helm_deploy/values-dev.yaml:4]: Does this ticket include the HPA stuff?

File: src/main/resources/banner.txt [hunk 4]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (nickmcmahon01) [inline src/main/resources/banner.txt:1]: no.

Comment (astroash) [reply src/main/resources/banner.txt:1]: Action based PR comments 🙏🏽

File: helm_deploy/values-dev.yaml (hunk 3 above)
Comment (astroash) [reply helm_deploy/values-dev.yaml:4]: No. As we're using the service template, this PR goes slightly beyond the scope of MAS-64

Comment (nickmcmahon01) [reply helm_deploy/values-dev.yaml:4]: > Keep the autoscaler, 1/2 for dev/preprod 2/4 for prod

File: src/main/resources/banner.txt (hunk 4 above)
Comment (nickmcmahon01) [reply src/main/resources/banner.txt:1]: we should run without a startup banner at all, as each line can end up in application insights.

**Review by nickmcmahon01 — APPROVED** [review]

**Pull request merged**